aerogear/mobile-security-service
# feat(app-detailed-view): Make a GET to the API to retrieve an app by its ID (AEROGEAR-8466)

## Motivation

https://issues.jboss.org/browse/AEROGEAR-8466

## What

Added logic to make a GET request to the API and fetch a single app by its ID.

## Why

It is required so that we the UI can fetch live, dynamic content from the server.

## How

Added new function to `DataService.js` and updated Redux store to handle new state changes.

## Verification Steps
 
1. Make sure the API is running.
2. On the landing page, click one of the app rows to bring you to the app detailed page.
3. Open the developer console and monitor the output from the redux-logger.
4. If you see a log with action type `APP_SUCCESS` then the UI has successfully retrieved the app information from the server.
5. Navigate to the app details view by clicking on one of the apps on the landing page.
6. The table of deployed versions should render and should be sortable on all columns.

## Checklist:


## Progress
